pkg/controller/daemon/daemoncontroller.go

```diff
@@ -89,6 +89,9 @@ type DaemonSetsController struct {
 	// Added as a member to the struct to allow injection for testing.
 	nodeStoreSynced cache.InformerSynced
 
+	// To allow injection of CreatePodTemplate for testing
```

Comment:
Yeah, I'm using this technique in daemon_util_test.go. I will change it here also